platform/src/abstractions/EducationBenchmarking.Platform.Infrastructure/Cosmos/DataGroups.cs

```diff
@@ -0,0 +1,14 @@
+namespace EducationBenchmarking.Platform.Infrastructure.Cosmos;
+
+public static class DataGroups
+{
+    public const string Edubase = "Edubase";
```

Comment:
Edubase is the older, redundant name for what the clients have now began calling "GIAS", which may cause confusion to devs without domain knowledge. Should we replace this before it is propagated it into the new service?

Reply:
Something we need to consider, if Edubase is a simple mirror of GIAS then it would make sense to change this.

Code impact is minimal, however we'd need to remove the fibre collection before committing to this change.